nixos/doc/manual/installation/installing.chapter.md

```diff
@@ -1,30 +1,127 @@
 # Installing NixOS {#sec-installation}
 
-## Booting the system {#sec-installation-booting}
+## Booting from the Install drive {#sec-installation-booting}
 
-NixOS can be installed on BIOS or UEFI systems. The procedure for a UEFI
-installation is by and large the same as a BIOS installation. The
-differences are mentioned in the steps that follow.
+To begin the installation, you have to boot your computer from the install drive.
+
+1.   Assuming your computer is still running, plug in the install drive
+and restart your computer.
+
+2.   Most computers will allow to select a drive to temporary boot from by
+pressing a special key, usually <kbd>F12</kbd>, but sometimes <kbd>Esc</kbd>
+or another function key. Refer to the screen or your computer's documentation.
+
+3.   Press <kbd>F12</kbd> (or the appropriate key) and select the install drive,
+usually "USB-HDD" or something containing the word "USB", but the wording may vary.
+If your computer support BIOS and UEFI boot, choose the UEFI option.
+If you choose the incorrect drive, your computer will likely continue to boot
+as normal. Just restart your computer and pick a different drive in that menu.
+
+4.   Shortly after selecting the appropriate boot drive, you should be presented
+with a menu with different Installer options. Leave the default and wait
+(or press <kbd>Enter</kbd> to speed up).
+
+5.   The graphical images will start the desktop environment and the graphical
+installer, which can take some time. The minimal images will just have
+a command line.
```

Comment:
```suggestion
installer, which can take some time. The minimal images will boot to
a command line.
```

The usage of *just* here is fine, but I'm proposing this alternate formulation that, I think, better describes what happens.

Reply:
good suggestion!